docs/src/param_retrieval.md

```diff
@@ -35,29 +74,28 @@ function ThermodynamicsParameters(toml_dict)
     # Used in thermodynamics, from parameter file
     aliases = [ ..., "gas_constant", "molmass_dryair"]
 
-    param_pairs = CLIMAParameters.get_parameter_values!(
+    parameters = CLIMAParameters.get_parameter_values!(
```

Comment:
remove "!"? 
maybe don't use the term _aliases-, maybe call them "names" now? or demo the name map?